catalog/pipeline/deploy-release/0.1/tests/run.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: release-run-empty-params
+spec:
+  params:
+    - name: snapshot
+      value: ""
+    - name: enterpriseContractPolicy
+      value: ""
+  pipelineRef:
+    name: deploy-release-pipeline
+    bundle: quay.io/hacbs-release/deploy-release:0.1
```

Comment:
I am quite sure this should be `pipeline-deploy-release` as our automation adds `pipeline-` and `task-`

Reply:
nice catch! updated.